_posts/2018-12-06.Rust-1.31.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: "Announcing Rust 1.31"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.31.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.31.0 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.31.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1310-2018-12-06
+
+## What's in 1.31.0 stable
+
+Rust 1.31 may be the most exciting release since Rust 1.0! Included in this release is the
+first iteration of "Rust 2018," but there's more than just that! This is going to be a long
```

Comment:
```suggestion
first iteration of "Rust 2018", but there's more than just that! This is going to be a long
```

Reply:
@Centril It's horrible yeah... I guess if we're going to be consistent though...